Implement WakeLock so that the system does not enter sleep (#2670)
dotnet/BenchmarkDotNet: docs/articles/samples/IntroWakeLock.md

---
uid: BenchmarkDotNet.Samples.IntroWakeLock
---

## Sample: IntroWakeLock

> Running Benchmarks usually takes enough time such that the system enters sleep or turns of the display.

Reviewer: ```suggestion
Running benchmarks may sometimes take enough time such that the system enters sleep or turns off the display.
```
